code/modules/mob/living/carbon/carbon_procs.dm

```diff
@@ -547,6 +549,8 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, list(/obj/machinery/atmospherics/unary/ven
 
 /mob/living/proc/remove_ventcrawl()
 	if(client)
+		our_pipeline.crawlers -= src
```

Comment:
`our_pipeline` is only used in one place, and we could use a signal here instead to remove ourself from the crawlers pipeline so we dont have to add another variable to every mob

Reply:
Switched it over to a signal, thanks for the tip.